src/components/EditCoursePage/index.jsx

```diff
@@ -193,7 +245,12 @@ class EditCoursePage extends React.Component {
       courseRunOptions,
       formValues,
     } = this.props;
-    const { startedFetching, isSubmittingForReview, targetRun } = this.state;
+    const {
+      isSubmittingForReview,
```

Comment:
Why pull this out? It looks like it isn't being used inside of this function.

Reply:
I did not add this.  it was preexisting, this is just a formatting edit and adding submitConfirmVisible.  All of these are used below.